tests/urls.py

```diff
@@ -0,0 +1,5 @@
+# from django.conf.urls import url
+
+urlpatterns = [
+    #  url(r'^admin/', admin.site.urls),
+]
```

Comment:
I'll actually just remove the module entirely since it's not being used.  I had intended to add some tests related to monkey-patching, but it's more complicated than I thought so we can rely on manual testing for now.